app/tests/src/test_format.py

```diff
@@ -71,6 +77,8 @@ def test__format_to_accordion_html(app_config, db_session, enable_factory_create
 
 def test_format_bem_documents():
     docs = DocumentFactory.build_batch(4)
+    for doc in docs:
+        doc.name += "BEM 123"
```

Comment:
Ensure that the doc names have a valid BEM number for link replacement by the formatter